docs/metrics.asciidoc

```diff
@@ -282,7 +282,7 @@ The number of transactions for which breakdown metrics (`span.self_time`) have b
 As the Go agent tracks the breakdown for both sampled and non-sampled transactions, this
 metric is equivalent to `transaction.duration.count`
 
-You can filter and group by these dimensions:
+Filter and group by these dimensions:
```

Comment:
I prefer "You can..." here because an imperative doesn't feel right here.

Reply:
```suggestion
You can filter and group by these dimensions:
```